rotkehlchen/externalapis/etherscan.py

```diff
@@ -548,6 +550,9 @@ def has_activity(self, account: ChecksumEvmAddress) -> EtherscanHasChainActivity
         result = self._query(module='account', action='tokentx', options=options)
         if len(result) != 0:
             return EtherscanHasChainActivity.TOKENS
+        balance = self._query(module='account', action='balance', options={'address': str(account)})  # noqa: E501
```

Comment:
Hmm @yabirgb this checks for all chains and not only ethereum mainnet. Does this make sense? Should we do it? Say in Gnosis ... does it take native asset (ETH?) from withdrawals/blocks? In polygon what is it? In the rollups there is no such concept. In other chains?

Reply:
ah true I forgot the other chains. If this is possible in ethereum it should be possible in gnosis as it the same process. In polygon I'm not aware how the staking works and in the rollups the concept doesn't exists so I would limit this query only to ethereum and gnosis